package.json

```diff
@@ -5,13 +5,14 @@
   "description": "A Strapi application",
   "license": "MIT",
   "scripts": {
-    "build": "strapi build",
+    "build": "npm run plugin:build && strapi build",
     "develop": "strapi develop",
-    "postinstall": "patch-package",
     "start": "strapi start",
     "strapi": "strapi",
     "test": "jest --forceExit --detectOpenHandles",
-    "transfer": "source .env && strapi transfer --from $TRANSFER_URL_REMOTE --from-token $TRANSFER_TOKEN_REMOTE --to-token $TRANSFER_TOKEN_LOCAL"
+    "transfer": "source .env && strapi transfer --from $TRANSFER_URL_REMOTE --from-token $TRANSFER_TOKEN_REMOTE --to-token $TRANSFER_TOKEN_LOCAL",
+    "plugin:build": "npm run build --workspace plugins/strapi-plugin-gh-workflow-trigger-v5",
```

Comment:
maybe give this a more verbose name like `custom-plugin` or `gh-workflow-trigger-plugin`